plugins/aws/managedblockchain/networkMemberCloudwatchLogs.js

```diff
@@ -0,0 +1,90 @@
+    realtime_triggers: ['managedblockchain:CreateNetwork', 'managedblockchain:DeleteMember'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        async.each(regions.managedblockchain, function(region, rcb){
+            var listNetworks = helpers.addSource(cache, source,
+                ['managedblockchain', 'listNetworks', region]);
+
+            if (!listNetworks) return rcb();
+
+            if (listNetworks.err || !listNetworks.data) {
+                helpers.addResult(results, 3,
+                    `Unable to query for Managed Blockchain networks: ${helpers.addError(listNetworks)}`, region);
+                return rcb();
+            }
+
+            if (!listNetworks.data.length) {
+                helpers.addResult(results, 0, 'No Managed Blockchain networks found', region);
+                return rcb();
+            }
+
+            for (let network of listNetworks.data) {
+                if (!network.Id || !network.Arn) continue;
+                
+                let listMembers = helpers.addSource(cache, source,
+                    ['managedblockchain', 'listMembers', region, network.Id]);
+
+                if (!listMembers || listMembers.err || !listMembers.data || !listMembers.data.Members) {
+                    helpers.addResult(results, 3,
+                        `Unable to query network members: ${helpers.addError(listMembers)}`,
+                        region, network.Arn);
+                    continue;
+                }
+
+                if (!listMembers.data.Members.length) {
+                    helpers.addResult(results, 0, 'No network members found', region, network.Arn);
+                    continue;
+                }
+
+                for (let member of listMembers.data.Members) {
+                    if (!member.Id || !member.Arn) continue;
+
+                    let resource = member.Arn;
+                    let getMember = helpers.addSource(cache, source,
+                        ['managedblockchain', 'getMember', region, member.Id]);
+    
+                    if (!getMember || getMember.err || !getMember.data || !getMember.data.Member) {
+                        helpers.addResult(results, 3,
+                            `Unable to query network member: ${helpers.addError(getMember)}`,
+                            region, member.Arn);
+                        continue;
+                    }
+                    const getmember = getMember.data.Member;
+
+                    if (getmember.LogPublishingConfiguration && getmember.LogPublishingConfiguration.Fabric &&
+                        getmember.LogPublishingConfiguration.Fabric.CaLogs && getmember.LogPublishingConfiguration.Fabric.CaLogs.Cloudwatch
+                        &&  getmember.LogPublishingConfiguration.Fabric.CaLogs.Cloudwatch.Enabled) {
```

Comment:
why are we just checking CA logs? what about Peer node logs , Chaincode logs ?

Reply:
we are only checking if the logging is enabled on the managedBlockchain member, Peer node logs , Chaincode logs comes under peer nods that are created in the members